rest_api/understanding-api-support-tiers.adoc

```diff
@@ -0,0 +1,16 @@
+[id="understanding-api-support-tiers"]
+= Understanding API support tiers
+include::modules/common-attributes.adoc[]
+:context: understanding-api-support-tiers
+
+toc::[]
+
+Support tiers for bare-metal configurations also apply to virtualized configurations except for any feature that directly interacts with hardware.  Those features directly related to hardware have no application operating environment (AOE) compatibility level beyond that which is provided by the hardware vendor.  For example, applications that rely on Graphics Processing Units (GPU) features are subject to the AOE compatibility provided by the GPU vendor driver.
```

Comment:
We should try an explicitly denote what API's in the product this is connected to. My guess is we don't have any but that 'third parties' might bring these to OpenShift. 

Do we want a way for those third parties to list them here?

Reply:
I assume we don't?